package.json

```diff
@@ -19,12 +20,15 @@
     "@types/node": "^16.7.13",
     "@types/react": "^18.0.0",
     "@types/react-dom": "^18.0.0",
+    "@types/react-input-mask": "^3.0.2",
+    "dayjs": "^1.11.7",
     "firebase": "^9.18.0",
     "i18n-js": "^4.2.3",
     "localforage": "^1.10.0",
     "match-sorter": "^6.3.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-imask": "^6.6.1-alpha.0",
```

Comment:
avoid using alpha version. I'll try to downgrade to the latest stable version (6.6.0). 

https://www.npmjs.com/package/react-imask?activeTab=versions